<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Person</class>
 <widget class="QWidget" name="Person">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Person</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/images/images/ico.ico</normaloff>:/images/images/ico.ico</iconset>
  </property>
  <widget class="QGroupBox" name="FinishedJob">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>671</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Finished Jobs</string>
   </property>
   <widget class="QTableWidget" name="FinishedJobTbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>631</width>
      <height>231</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>58</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Job Name</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Join Time</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Starting</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Finish</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last Time</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Deadline</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Turn Over</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weight Turn over</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ATTHL">
     <item>
      <widget class="QLabel" name="ATT">
       <property name="text">
        <string>Average Turnover Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ATTValue">
       <property name="text">
        <string>time</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="WATTHL">
     <item>
      <widget class="QLabel" name="WATT">
       <property name="text">
        <string>Weighted Average Turnover Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="WATTValue">
       <property name="text">
        <string>time</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="PreInput">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>671</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Failed Jobs</string>
   </property>
   <widget class="QTableWidget" name="FailedJobTbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>541</width>
      <height>171</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>65</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>15</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Job Name</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Join Time</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Starting</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Finish</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last Time</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DeadLine</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="FailLbl">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>81</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/images/fail.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="personBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Person Information</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>214</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="UserNameLbl">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>User Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UserNameLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="PasswdLbl">
         <property name="font">
          <font>
           <family>Calibri</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Password   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PasswdLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ModifyBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(249, 249, 249)</string>
    </property>
    <property name="text">
     <string>Modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(249, 249, 249)</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="WarningChkBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>154</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Warning</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="EchoChkBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>123</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Echo</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="graphBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Graph</string>
   </property>
   <widget class="QLabel" name="GraphLbl">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Click to set graph</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gifBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Gif</string>
   </property>
   <widget class="QLabel" name="GifLbl">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>21</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Click to set gif</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>561</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/images/images/Urban Dream.jpg)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>FinishedJob</zorder>
  <zorder>PreInput</zorder>
  <zorder>personBox</zorder>
  <zorder>graphBox</zorder>
  <zorder>gifBox</zorder>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
